modular_skyrat/modules/reagent_forging/code/tool_override.dm

```diff
@@ -0,0 +1,104 @@
+/// Called on an object when a tool with wrench capabilities is used to left click an object
+/atom/proc/billow_act(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to right click an object
+/atom/proc/billow_act_secondary(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to left click an object
+/atom/proc/tong_act(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to right click an object
+/atom/proc/tong_act_secondary(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to left click an object
+/atom/proc/hammer_act(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to right click an object
+/atom/proc/hammer_act_secondary(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to left click an object
+/atom/proc/blowrod_act(mob/living/user, obj/item/tool)
+	return
+
+/// Called on an object when a tool with wrench capabilities is used to right click an object
+/atom/proc/blowrod_act_secondary(mob/living/user, obj/item/tool)
+	return
+
+/**
+ *Tool behavior procedure. Redirects to tool-specific procs by default.
+ *
+ * You can override it to catch all tool interactions, for use in complex deconstruction procs.
+ *
+ * Must return  parent proc ..() in the end if overridden
+ */
+/atom/tool_act(mob/living/user, obj/item/tool, tool_type, is_right_clicking)
+	var/act_result
+	var/signal_result
+	if(!is_right_clicking) // Left click first for sensibility
```

Comment:
you could probably ternerize all of the `act_result = TOOL_act(user, tool)` to reduce on code duplication with one-word difference

Reply:
its a default /tg/ proc... I don't really want to touch it unless I touch it upstream

Reply:
Fair enough